src/compliance/privacy/ccpa.md

```diff
@@ -0,0 +1,43 @@
+---
+group: compliance
+title: California Consumer Privacy Act
+---
+
+The [California Consumer Protection Act][1]{: target="_blank"} (CCPA) expands the rights of consumers in California to determine how their personal information is collected, stored, and used, with an emphasis on protecting consumers from the unauthorized sale or exchange or their personal information. The CCPA was enacted in 2018 and is effective as of January 1, 2020.
+
+The CCPA grants the following new rights to consumers:
+
+-  **Right to know** the categories of personal information about them that is collected, used, shared, or sold in the past 12 months.
+-  **Right to delete** certain types of personal information that is held by a business and/or their service provider(s).
+-  **Right to opt out** of the sale of their personal information.
+-  **Right to non-discrimination** in terms of price or service for having exercised a privacy right under CCPA.
+
+## CCPA Compliance Guide
+
+A coordinated effort is required to develop and implement a plan to address CCPA compliance. Merchants are encouraged to assemble a cross-functional team, and follow the roadmap outlined in [CCPA Compliance Guide][2]{: target="_blank"} to bring their company into compliance with the regulation. As a developer, you might be invited to participate as a stakeholder with an emphasis on steps 2 - 5 of the process. See the CCPA Compliance Guide for more information.
```

Comment:
```suggestion
Developing and implementing a CCPA compliance plan requires a coordinated effort. We encourage merchants to assemble a cross-functional team, and follow the roadmap outlined in [CCPA Compliance Guide][2]{: target="_blank"} to bring their company into compliance with the regulation. As a developer, you might be invited to participate as a stakeholder with an emphasis on steps 2 - 5 of the process. See the CCPA Compliance Guide for more information.
```

Reply:
Should "CCPA Compliance Guide" have a link reference?

Reply:
Thanks for the edit!  As for the link, I didn't link the second reference to the CCPA Compliance Guide, because the first reference is already linked.  But it *feels* like it should be linked, so I will.

Reply:
Delete the `{: target="_blank"}`

Reply:
Fixed.  (Actually, I had it correct initially, but added the 'target = blank' because it didn't seem to be opening in a new tab -- might have been something weird on my part) Thanks!